.github/workflows/main.yml

```diff
@@ -0,0 +1,19 @@
+name: Docker Image CI
```

Comment:
Why is this file needed?

Reply:
Not sure why , it was created upon executing make deps-update. I'll try to redo the process .

Reply:
After doing the process again, I see that the main.yaml gets created & added & committed automatically upon "make deps-update". Should I try & revert this file commit? @MarSik  please advise.

Reply:
ci tests passed without the main.yaml, will close this one since https://github.com/openshift-kni/cnf-features-deploy/pull/680 already passed.